### PR TITLE
sql_variant and sysname as string

### PR DESCRIPTION
**Pull request by hounded**

### CODE REVIEW

File: lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php [hunk 1]
```diff
@@ -1406,6 +1406,8 @@ protected function initializeDoctrineTypeMappings()
             'varbinary' => 'binary',
             'image' => 'blob',
             'uniqueidentifier' => 'guid',
+            'sql_variant'=>'string',
```
Comment (phansys) [inline lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php]: Please, respect CS (missing spaces around `=>` operator).

**Review by phansys — CHANGES_REQUESTED** [review]
Status: Needs Work.

**Comment by hounded** [comment]
Respected CS (missing spaces around => operator).

**Comment by Ocramius** [comment]
Seems straightforward, but a unit test is needed to prevent future regressions.

**Comment by federicoemartinez** [comment]
Is there a workaround if this PR is not merged? I have already faced an issue with "sysname"

**Comment by morozov** [comment]
Closing as SQLAnywhere is no longer supported.

**Pull request closed without merging**